### PR TITLE
dioxus-desktop(Update-dep): tauri 0.19.x

### PR DESCRIPTION
**Pull request by Ar37-rs**

### CODE REVIEW

File: packages/desktop/src/lib.rs [hunk 1]
```diff
@@ -213,12 +213,14 @@ pub fn launch_with_props<P: 'static + Send>(
...
-                    }
-                }
+                // WindowEvent::Resized(_) | WindowEvent::Moved(_) => {
+                //     if let Some(view) = desktop.webviews.get_mut(&window_id) {
+                //         let _ = view.resize();
+                //     }
+                // }
+                //
+                // No longer required manually resizing,
+                // see: https://github.com/tauri-apps/wry/commit/d7c9097256d76de7400032cf27acd7a1874da5cd
```
Comment (rMazeiks) [inline packages/desktop/src/lib.rs:223]: that's cool! i think we can just delete these lines then, no need to leave them commented out

Comment (Ar37-rs) [reply packages/desktop/src/lib.rs:223]: [rMazeiks](https://github.com/rMazeiks) Ok

Comment (Ar37-rs) [reply packages/desktop/src/lib.rs:223]: @rMazeiks oh ok.

**Review by rMazeiks — APPROVED** [review]
Thanks!!

**Review by jkelleyrtp — APPROVED** [review]

**Comment by jkelleyrtp** [comment]
Sorry - can you format this branch so checks pass?

**Comment by Ar37-rs** [comment]
@jkelleyrtp done

**Comment by Ar37-rs** [comment]
@jkelleyrtp oh my, sorry i'm online on github mobile right now so i can't cargo fmt directly. it's the last fixing, hopefully everything will be fixed thank you

**Pull request merged**